jingo_minify/management/commands/compress_assets.py

```diff
@@ -39,7 +37,8 @@ class Command(BaseCommand):  # pragma: no cover
 
     def update_hashes(self, update=False):
         def media_git_id(media_path):
-            id = git.repo.Repo(path(media_path)).log('-1')[0].id_abbrev
+            id = check_output(["git", "-C", path(media_path),
```

Comment:
Might be worth adding a `GIT_BIN` setting that defaults to `git` so that it's an easy fix if git isn't in the server PATH.